.devcontainer/Dockerfile

```diff
@@ -1,16 +1,25 @@
-FROM mcr.microsoft.com/devcontainers/miniconda:0-3
+FROM python:3.10
 
-# Copy environment.yml (if found) to a temp location so we update the environment. Also
-# copy "noop.txt" so the COPY instruction does not fail if no environment.yml exists.
-COPY environment.yml* .devcontainer/noop.txt /tmp/conda-tmp/
-RUN if [ -f "/tmp/conda-tmp/environment.yml" ]; then umask 0002 && /opt/conda/bin/conda env update -n base -f /tmp/conda-tmp/environment.yml; fi \
-    && rm -rf /tmp/conda-tmp
+# Set up the working directory.
+WORKDIR /code
+COPY ./ /code
 
-# [Optional] Uncomment to install a different version of Python than the default
-# RUN conda install -y python=3.6 \
-#     && pip install --no-cache-dir pipx \
-#     && pipx reinstall-all
+# RUN npm install
+RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
+     && apt-get -y install --no-install-recommends npm
+RUN npm install
 
-# [Optional] Uncomment this section to install additional OS packages.
-# RUN apt-get update && export DEBIAN_FRONTEND=noninteractive \
-#     && apt-get -y install --no-install-recommends <your-package-list-here>
+# Update pip
+RUN pip install --upgrade pip
+# Install packages
+RUN pip install torch torchvision torchaudio tensorboardX
+RUN pip install -U pip setuptools wheel
```

Comment:
Is this redundant? Maybe we should do the pip/setuptools/wheel update once, before using pip to install anything else.